app/views/capture.njk

```diff
@@ -0,0 +1,80 @@
+{% extends '_layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "macros/search-box.njk" import searchBox %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+        text: "Back",
+        href: "/",
+        attributes: {
+          id: "back"
+        }
+      }) 
+    }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l">Capture reporting data</h1>
+      <p class="govuk-body">
+        Datasets are automatically attached to matching requests when they are received in the request editor.
+      </p>    
+      <a href="/capture-new" class="govuk-button govuk-button--start">
```

Comment:
Discretion I got once but and don't know I can replicate but the button appeared different: smaller sizes and smaller font size

Looked at element and its class was taken from `_index.scss` rather than `core` as seen in the prototype -- re-ran today and no issue

Thought I'd drop a comment about it

Reply:
Gave this a go a few times and I can't replicate :(